Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml

```diff
@@ -174,7 +174,7 @@
   parent: EncryptionKey
   id: EncryptionKeySecurity
   name: security encryption key
-  description: An encryption key used by security.
+  description: An encryption key used by security. Maybe it has additional encryption?
```

Comment:
What is the point of hinting additional encryption if this works like any other key?

Reply:
that's a sec key, I think it would be nice to have the same "easter egg" as in the other keys